### PR TITLE
Environment Variable Addition

### PR DESCRIPTION
**Pull request by LinkExe**
-When logging with LogEventInfo, you can now set properties on the
LogEventInfo and they will display on the Environment Tab on Errbit.

### CODE REVIEW

File: src/NLog.AirBrake/AirBrakeTarget.cs [hunk 1]
```diff
@@ -44,6 +45,18 @@ protected override void Write(LogEventInfo logEvent)
         string exceptionMessage = BuildExceptionMessage(logEvent.Exception);
         notice.Error.Message = !string.IsNullOrEmpty(logEvent.FormattedMessage) ? logEvent.FormattedMessage + " " + exceptionMessage : exceptionMessage;
 
+        //Any LogEventInfo.Properties will be added to the Environment Tab.
+        var environmentProperties = new List<AirbrakeVar>();
```
Comment (Iristyle) [inline src/NLog.AirBrake/AirBrakeTarget.cs]: Looks great.. just one minor thing.  You can initialize the list with the existing variables

``` csharp
var environmentProperties = new List<AirbrakeVar>(notice.Request.CgiData);
```

File: src/NLog.AirBrake/AirBrakeTarget.cs [hunk 2]
```diff
@@ -44,6 +45,18 @@ protected override void Write(LogEventInfo logEvent)
...
+        foreach (var property in logEvent.Properties)
+        {
+            AirbrakeVar prop = new AirbrakeVar(property.Key.ToString(), property.Value);
+            environmentProperties.Add(prop);
+        }
+        foreach (AirbrakeVar cgiData in notice.Request.CgiData)
```
Comment (Iristyle) [inline src/NLog.AirBrake/AirBrakeTarget.cs]: And then delete this `foreach`...

**Comment by LinkExe** [comment]
Updated with your suggestions, thank you.

**Comment by Iristyle** [comment]
Cool, less lines = mo betta`

Good job!

**Pull request merged**